app/views/pregnancies/_patient_dashboard.html.erb

```diff
@@ -1,3 +1,7 @@
+<div class="col-md-12">
```

Comment:
Can we ditch this div? I think just having the render is fine, and it doesn't seem to make a difference at the end?